src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Concurrent;
 using System.IO;
 using System.Linq;
+using System.Security.Cryptography.Xml;
```

Comment:
???

Reply:
This is visual studio autonamespace complete thingy that leaves garbage all over the place. I'll remove.